source/Preferences.cpp

```diff
@@ -145,6 +152,27 @@ string Preferences::AmmoUsage()
 }
 
 
+// FIXME: date format change refuses to function properly
+void Preferences::ToggleDateFormat()
```

Comment:
You create this function but you have no place where it is called. Could be the problem

Reply:
You need something like this PreferencesPanel.cpp:
`else if(zone.Value() == EXPEND_AMMO)
				Preferences::ToggleAmmoUsage();`

Reply:
I actually called the function to toggle it. Now it works!